rustworkx/visualization/matplotlib.py

```diff
@@ -754,6 +754,16 @@ def _connectionstyle(posA, posB, *args, **kwargs):
         else:
             line_width = width
 
+        # radius of edges
+
+        reverse_edge = np.concatenate(([dst], [src]))
+        for edge in edge_pos:  # the loop can be optimized
+            if bool(np.sum(np.all(np.equal(edge, reverse_edge)))):
```

Comment:
I think there are easier and simpler ways to write this. I don't quite understand why we are using NumPy for this check either